build

```diff
@@ -82,21 +112,27 @@ build_test_all() {
     build_deps
     build_maude
     build_tangle
-    make check "$@"
+    make -s check "$@"
     git checkout -- src/Mixfix/tokenizer.cc
+    build_maude_cvc4
+    build_test tests/Misc/smtTest
+    build_maude_yices2
+    build_test tests/Misc/smtTest
 }
 
 # Main
 # ----
 
 build_command="$1" ; shift
 case "$build_command" in
-    clean)    build_clean    "$@" ;;
-    deps)     build_deps     "$@" ;;
-    maude)    build_maude    "$@" ;;
-    tangle)   build_tangle   "$@" ;;
-    test)     build_test     "$@" ;;
-    test-all) build_test_all "$@" ;;
+    clean)          build_clean    "$@" ;;
+    deps)           build_deps     "$@" ;;
+    maude)          build_maude    "$@" ;;
+    maude-cvc4)     build_maude_cvc4 "$@" ;;
+    maude-yices2)   build_maude_yices2 "$@" ;;
+    tangle)         build_tangle   "$@" ;;
+    test)           build_test     "$@" ;;
+    test-all)       build_test_all "$@" ;;
```

Comment:
Please line up the `$@` and make sure that all options here are documented below.